### PR TITLE
Update Alpha wallet logo & uri

### PR DESCRIPTION
**Pull request by cloakd**
Adjust wallet logo & uri to reflect github change (babilu-online -> alphabatem)

### CODE REVIEW

File: packages/wallets/alpha/src/adapter.ts [hunk 1]
```diff
@@ -46,13 +46,12 @@ declare const window: AlphaWindow;
 
 export interface AlphaWalletAdapterConfig {}
 
-export const AlphaWalletName = 'Alpha' as WalletName<'Alpha'>;
+export const AlphaWalletName = 'Alpha Wallet' as WalletName<'Alpha Wallet'>;
```
Comment (jordansexton) [inline packages/wallets/alpha/src/adapter.ts:49]: In general, we have avoided superfluous use of `Wallet` in wallet names unless the wallet predated Wallet Adapter itself.

File: packages/wallets/alpha/src/adapter.ts [hunk 2]
```diff
@@ -46,13 +46,12 @@ declare const window: AlphaWindow;
...
...L+ShyfYYYQP0b/CtdtiTcDGiF05rbDabt7u6DzURZTo1GIFFrgq93FTUfiWgWQAV+iiVr7h0E3OLXIgHTs5OIj4Kdvnc5BhEtfpZp/4HzdonncdOSWGQDnaeEzm1lCOl11nkHrELt135iSXWJb+MRyywxAAISFD+cyZBwC8A8zXSDgp4PWBREWUqVKztGLdS/MwDXp/JQjjhkA5+mD/QXMErLDVecphkFT27aKrNIKNsTrPMUxAxCAN9GsZZYQWoScZWrKBHHhoi10ND9vO4xNxyyCrXgshbLW8/34prvqs4vjfvNsGwA4T9/k0gnOae1PJ/2QagThIjMF/LT1XmKQYwgApnpcOnaiw5mg+Vaerl24lZtx4XQLLIa45TyFvkhczlQbQno70ZHdBpXoVw05khJsGTVxJzfjRpgBGH1a+HRDJpgn3IP+eKIGWXCdmcM43llFQuFTSH2NKDQDuPAsKFfz3uite/4100gAuwygDzPddp5yB+RZpsZwY3AFRl8jOkZ+uOCfKsn9Dbll9HFbwtgFoIO3XmB9bX68swDpv4w5PMx509aJT9Y+2jN2Ti/vnRCWACD9Z6J5gFmeMBXXEL3DuzawGKNfPOQ0c5zOB1jw9BqlIOlS3DIH4MtfRvMIsyzgm83iJx3MhYxhqgW6tX7S1H55VyHBy1240jvNe95cAYakQM3af2VK6xNm3yQQToJugEC/imY9syzQQmYigtBDDq9vIoOhL4Y7bQoPBAqfWefuXnrr+sEG/BXP+ABit12l17XK1G6E2i33PZdsQzqaivMUzwKA0aUPLJy2rC0zftw9G6M/03TYMvkZJEeX3uZ9k8bLDKBsgdjV7YWN+ZWfRjvNiyBMfqpunGqUSh2f98WDpwFAFtBi6yCzrLxYNbVIpdMUrf5MGRp9VSeb9ty3wPF5Xzx4nQEU2zdF1blFpKlskiX9Jd34s1RXEy57RYyEAHwNOcFUK20TGm6OeiQQim5s/+3Btqu8S0p4HgDcBvDKPgsaSqrI9KLKoQwwjAFNlxLa8TkxEjKA8gnEtpRtj2SBWfkZu/sfXUt/6ZEWRkQAkAV092f58UKEporJZFyOjwZBlw1PnkDfflAZjoH0Q4Ss7z5ikH3rnN4B/veBn52QXpEEr1/trT600ek1WIYMGTJkyJAhEQj5Fz7jUetUp2rXAAAAAElFTkSuQmCC';
```
Comment (jordansexton) [inline packages/wallets/alpha/src/adapter.ts:54]: How much larger in bytes is this base64 PNG than the base64 SVG?

Comment (cloakd) [reply packages/wallets/alpha/src/adapter.ts]: Updated back to SVG

**Comment by cloakd** [comment]
Apologies on the delay getting back to this, was focused on getting it onto google store, should be ok now!

**Comment by jordansexton** [comment]
Published @solana/wallet-adapter-alpha@0.1.4

File: packages/wallets/alpha/src/adapter.ts [hunk 3]
```diff
@@ -109,7 +107,7 @@ export class AlphaWalletAdapter extends BaseMessageSignerWalletAdapter {
 
             if (!wallet.isConnected) {
                 try {
-                    await wallet.connect();
+                    return await wallet.connect();
```
Comment (jordansexton) [inline packages/wallets/alpha/src/adapter.ts:110]: I fixed this in master, but this should not return here. It breaks the `connect` function of the adapter, which is expected to dispatch events and do setup at the end.

**Pull request merged**